easyaudit/signals/model_signals.py

```diff
@@ -71,7 +80,54 @@ def post_save(sender, instance, created, raw, using, update_fields, **kwargs):
                                     for callback in CRUD_DIFFERENCE_CALLBACKS if callable(callback))
 
             # create crud event only if all callbacks returned True
-            if create_crud_event:
+            if create_crud_event and not created:
```

Comment:
Ah, I see what you've done.

On pre-save, you add in the changed_fields, if we're an edit, not a creation.

And on post-save, you save a new crud event if the object is newly created (and thus no changed_fields).

Curious, why could all of this logic not be done in the post_save handler? As in, why make the two cases completely separate like this, handler wise?